templates/devfile.yaml

```diff
@@ -0,0 +1,83 @@
+schemaVersion: 2.0.0
+metadata:
+  name: java-openliberty
+  version: 0.4.1
+  description: Java application stack using Open Liberty runtime
+  alpha.build-dockerfile: "{{.DEVFILE_DOCKERFILE_LOC}}"
+  alpha.deployment-manifest: "{{.DEVFILE_DEPLOY_YAML_LOC}}"
```

Comment:
Debatable where to add the old outer loop syntax to new stuff.. I think it's helpful on balance so leave it ..looks good.